build_json.mk

```diff
@@ -0,0 +1,27 @@
+# Look for a json keymap file
+ifneq ("$(wildcard $(MAIN_KEYMAP_PATH_5)/keymap.json)","")
```

Comment:
can you use a loop against a these wildcard paths?

Reply:
Make is a pretty limited language. This is the pattern we use several other places.